blob/blob.go

```diff
@@ -1042,7 +1042,27 @@ func (mux *URLMux) OpenBucketURL(ctx context.Context, u *url.URL) (*Bucket, erro
 	if err != nil {
 		return nil, err
 	}
-	return opener.(BucketURLOpener).OpenBucketURL(ctx, u)
+	return applyPrefixParam(ctx, opener.(BucketURLOpener), u)
```

Comment:
We should definitely document this query parameter.

Reply:
Done.

Reply:
(by adding a docstring to `blob.OpenBucket`, and by adding an example).

I'll update the HOWTO on `gocloud.dev` separately.